scss/tech.scss

```diff
@@ -0,0 +1,50 @@
+.tech-background {
+    background-clip: content-box;
+    background-color: $white;
+    border-radius: 5px;
+    border: 1px solid $green;
+    margin-top: 10px;
+}
+
+.modify-img {
+    padding-top: 5px;
+}
+
+.size-tech {
+    height: 340px;
+    width: 220px;
+}
+
+.subtitle {
+    padding-bottom: 15px;
+}
+
+.tech {
+    margin-top: 10px;
+    margin-left: 10px;
+    background: $green;
+    border-radius: 5px;
+}
+
+.order-tech {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
```

Comment:
Use 'space-around' and 'space-between' instead 'space-evenly' and use margins to fix the position.